### ISSUE TITLE
Problem with run the project

### ISSUE THREAD

**Issue by CainSamael** [BoBeR182/PoGoBoWo#38]
When i running the bot i have these problems in the CMD
Traceback (most recent call last):
  File "pokebot.py", line 15, in <module>
    from pgoapi import PGoApi
  File "C:\Users\Daniel\PoGoBoWo\pgoapi\__init__.py", line 18, in <module>
    from pgoapi.pgoapi import PGoApi # noqa
  File "C:\Users\Daniel\PoGoBoWo\pgoapi\pgoapi.py", line 21, in <module>
    from pgoapi.location import distance_in_meters, get_increments, get_neighbor
s, get_route, filtered_forts
  File "C:\Users\Daniel\PoGoBoWo\pgoapi\location.py", line 5, in <module>
    from gmaps.errors import GmapException
ImportError: No module named gmaps.errors

**Comment by BoBeR182** [comment]
What OS are you on? and commit are you on? May you post details to replicate the steps.

Have you done the `pip install -r requirements.txt` ?

**Comment by CainSamael** [comment]
i have windows 10 and yes i done the pip install -r requirements.txt

**Comment by BoBeR182** [comment]
What commit are you running, is python in your PATH?

**Comment by CainSamael** [comment]
yes python is in my path, a commit? what is it? my english isnt good so can you help me with that

**Comment by BoBeR182** [comment]
https://github.com/BoBeR182/PoGoBoWo/commits/master

`git pull` will update you to the latest commit, to make sure you are not running anything outdated.

Please also attach your `config.json` without any personal information.

**Comment by CainSamael** [comment]
[config.txt](https://github.com/BoBeR182/PoGoBoWo/files/391508/config.txt)

**Comment by CainSamael** [comment]
And i use a master commit

**Comment by buriedsand** [comment]
I wonder if I had the same problem as you? For some reason, most Windows can't just simply "pip install pyproj" which is a module in the requirements.txt file, so when I tried running "pip install -r requirements.txt" I would get an error.

Here's how I installed pyproj and the rest of the requirements.txt:

1. Download "pyproj-1.9.5-cp27-none-win32.whl"  http://www.lfd.uci.edu/~gohlke/pythonlibs/#pyproj 

2. Open Command Prompt and enter: C:\Python27\Scripts\pip install pyproj-1.9.5-cp27-none-win32.whl

3. Open Finder Explorer, navigate to C:\Users\ADMIN\PoGoBoWo (change ADMIN to your appropriate name) and open the "requirements" txt file

4. Delete the line that says "pyproj" and save

5. Open Command Prompt, enter: cd PoGoBoWo

6. enter: C:\Python27\Scripts\pip install -r requirements.txt
